ui-tests/cypress/e2e/administration/language_and_labels/system_labels.cy.js

```diff
@@ -363,6 +383,31 @@ describe('Labels', () => {
     });
 
   });
+  const getKeysFromLabelsJSON = () => {
+    cy.fixture('data/labels.json').then(labels => Object.values(labels).map(label => label.key)).as('keysFromJSON');
+    return cy.get('@keysFromJSON');
+  };
+
+  const generateDataFromJsonWithRandomCharacter = () => {
+    getKeysFromLabelsJSON().then(labels => {
+      let filteredLabels;
+      let randomChar;
+      do {
+        let char       = generateRandomString(1);
+        randomChar     = char;
+        filteredLabels = labels.filter(label => label.includes(char));
+      } while (filteredLabels.length < 11);
+      cy.then(() => [randomChar, filteredLabels]).as('dataReturned');
```

Comment:
If you return an object, the meaning of the computed value will be more clear when used it in the calling function